pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
 	core_v1 "k8s.io/api/core/v1"
```

Comment:
```GO
	// +optional
	ReadyToUse bool `json:"readyToUse" protobuf:"varint,3,opt,name=readyToUse"`		ReadyToUse bool `json:"readyToUse" protobuf:"varint,3,opt,name=readyToUse"`
```
@thockin: Either make this a pointer or make it required.

Consider the pre-provisioned case too. Most likely it should be optional -- so you can differentiate between never set (`nil`) and true/false.

Reply:
P0